pkg/k8s/cnp.go

```diff
@@ -124,6 +124,7 @@ func (c *CNPStatusUpdateContext) prepareUpdate(cnp *types.SlimCNP, scopedLog *lo
 		serverRule = localCopy.DeepCopy()
 		_, err = serverRule.Parse()
 		if err != nil {
+			err = ErrParse{err.Error()}
```

Comment:
What is the significance of this being a parse error vs. not being one?

Reply:
If it's parse error it'll updated the status in the CNP if not, no.